OpenStreetMapViewer/src/main/java/org/osmdroid/samplefragments/data/SampleWithMinimapItemizedOverlayWithScale.java

```diff
@@ -62,7 +62,7 @@ protected void addOverlays() {
 		{
 			/* Create a static ItemizedOverlay showing some Markers on various cities. */
 			final ArrayList<OverlayItem> items = new ArrayList<>();
-               for (int i=0; i < 500; i++){
+               for (int i=0; i < 5000; i++){
```

Comment:
What are you trying to test it with 5000 ?

Reply:
I test with 5k in order to check that we get the best performances even with a large crowd of markers. The point being to dismiss the markers that won't be displayed.
I assumed it was more or less the purpose of this demo.

Reply:
You're right, it's written in your commit message.